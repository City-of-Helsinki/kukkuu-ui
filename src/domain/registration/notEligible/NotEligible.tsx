import React, { FunctionComponent } from 'react';
import { useTranslation } from 'react-i18next';
import { Button } from 'hds-react';

import styles from './notEligible.module.scss';
import adultFaceIcon from '../../../assets/icons/svg/adultFace.svg';
import Icon from '../../../common/components/icon/Icon';
import PageWrapper from '../../app/layout/PageWrapper';

const NotEligible: FunctionComponent = () => {
  const { t } = useTranslation();

  return (
    <PageWrapper title={t('registration.notEligible.title')}>
      <div>
        <div className={styles.notEligible}>
          <Icon className={styles.notEligibleFace} src={adultFaceIcon} />
          <p>{t('registration.notEligible.text')}</p>
          <Button
            className={styles.goBackButton}
            onClick={() =>
              (window.location.href = t(
                'registration.notEligible.otherOptionsLink'
              ))
            }
          >
            {t('registration.notEligible.buttonText')}
          </Button>
        </div>
      </div>
    </PageWrapper>
  );
};

export default NotEligible;
