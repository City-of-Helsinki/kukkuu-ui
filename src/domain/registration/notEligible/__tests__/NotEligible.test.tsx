import { customRender as render } from '../../../../common/test/customRender';
import NotEligible from '../NotEligible';

it('renders snapshot correctly', () => {
  const { container } = render(<NotEligible />);
  expect(container).toMatchSnapshot();
});
