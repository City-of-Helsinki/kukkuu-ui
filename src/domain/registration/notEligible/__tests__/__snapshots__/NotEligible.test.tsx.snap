// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot correctly 1`] = `
<PageWrapper
  title="Voi harmi!"
>
  <div>
    <div
      className="notEligible"
    >
      <Icon
        className="notEligibleFace"
        src="adultFace.svg"
      />
      <p>
        Valitettavasti lapsen syntymävuosi tai kotipaikkakunta eivät oikeuta häntä kulttuurin kummilapset toimintaan.
      </p>
      <ForwardRef
        className="goBackButton"
        onClick={[Function]}
      >
        Löydä lapsille sopivia kulttuuritapahtumia täältä
      </ForwardRef>
    </div>
  </div>
</PageWrapper>
`;
